src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -95,6 +97,7 @@
  */
 class OperationsGapicClient
 {
+
```

Comment:
https://github.com/GoogleCloudPlatform/google-cloud-php/pull/715#discussion_r147433751

Reply:
fixing cs php-cs-fixer in artman fixed the problem ([artman PR](https://github.com/googleapis/artman/pull/310))